.github/workflows/check_code_style.yml

```diff
@@ -10,10 +10,18 @@ on:
     branches:
       - "main"
     paths-ignore:
-      - "**.md" 
+      - "**.md"
   pull_request:
     paths-ignore:
-      - "**.md"       
+      - "**.md"
+  # Optionally use tmate to live debug failures:
+  # https://github.com/mxschmitt/action-tmate#manually-triggered-debug
+  workflow_dispatch:
+    inputs:
+      debug_enabled:
+        description: 'Run the build with tmate debugging enabled (https://github.com/marketplace/actions/debugging-with-tmate)'
+        required: false
+        default: false
```

Comment:
Actually, how about we remove support for debugging here all together!? In my experience thus far any failures are printed out and we basically just need to go (re)run `clang-format` or `buildifier` so seems like we can simplify here and remove all together.

Reply:
Sold: removed.